storm-core/src/jvm/backtype/storm/Config.java

```diff
@@ -1215,6 +1254,7 @@
      * to be equal to the number of workers configured for this topology. If this variable is set to 0,
      * event logging will be disabled.</p>
      */
+    @isPositiveNumber
```

Comment:
This is a really minor nit, but everywhere else `@isInteger` is above `@isPositiveNumber` it would be nice to be consistent.